### PR TITLE
Remove vagrant-specific terms or steps from TP tutorial

### PR DESCRIPTION
**Pull request by ojalatodd**
Some vagrant-specific terms remained in the txn processor
tutorial. This commit removes remaing vagrant-specific
terms or steps.

Signed-off-by: Todd Ojala <todd@bitwise.io>

### CODE REVIEW

File: docs/source/_templates/sdk_TP_tutorial.rst [hunk 1]
```diff
@@ -44,9 +44,9 @@ Prerequisites
 This tutorial assumes that you have gone through :doc:`/app_developers_guide/intro_to_sawtooth` and are
 familiar with the concepts introduced there.
 
-Prior to going through this tutorial, you should have a working Vagrant
-environment running to which you can login.  Specific setup instructions are
-available in :doc:`/app_developers_guide/environment_setup`.
+Prior to going through this tutorial, you should have a working application
```
Comment (aludvik) [inline docs/source/_templates/sdk_TP_tutorial.rst]: Remove "to which you can login" as there won't always be a need to "login" to the environment.

File: docs/source/_templates/sdk_TP_tutorial.rst [hunk 2]
```diff
@@ -44,9 +44,9 @@ Prerequisites
...
+developer environment running to which you can login.  Specific setup
```
Comment (vaporos) [inline docs/source/_templates/sdk_TP_tutorial.rst]: Replace "application developer environment" with something like "Sawtooth environment".

File: docs/source/_templates/sdk_TP_tutorial.rst (hunk 1 above)
Comment (ojalatodd) [inline docs/source/_templates/sdk_TP_tutorial.rst]: Removed.

File: docs/source/_templates/sdk_TP_tutorial.rst (hunk 2 above)
Comment (ojalatodd) [inline docs/source/_templates/sdk_TP_tutorial.rst]: I like it.

**Pull request merged**